.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # This should match the owning team set up in https://github.com/orgs/opensearch-project/teams @opensearch-project/common-utils
-* @lezzago @qreshi @bowenlan-amzn @rishabhmaurya @getsaurabh02 @eirsep @sbcd90 @AWSHurneyt @engechas
+* @lezzago @qreshi @bowenlan-amzn @rishabhmaurya @getsaurabh02 @eirsep @sbcd90 @AWSHurneyt @engecha @riysaxen-amzn
```

Comment:
Is it a spelling mistake here?

Reply:
thanks for noticing, i will correct this

Reply:
addressed